### PR TITLE
Code Refactoring of App.js and Logic change in Home.js

### PR DESCRIPTION
**Pull request by sirichandanagangam**
# Proposed Changes

Shifted Logic from App.js to Home.js and changed logic in Home.js ( Changed code to for screens which has multiple buttons)

# Jira Stories
[LAB-248] 

# Checklist
[ ] This has been tested
[ ] Self-documenting code
[ ] Code is linted

### CODE REVIEW

**Review by radgupte — COMMENTED** [review]
On first look, the code looks fine, but I see you have linting errors in the build, which is causing it to fail. You can find those in the run-lint under the checks tab and resolve them.

File: src/HomePage/Home.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import { getFirestore, doc, updateDoc } from 'firebase/firestore'
+import { app } from '../Fire'
+
+const data = [
+  {
+    current: 1,
+    one: 2,
+    two: 9,
+    path: 'images/screen1.png'
+  }, {
```
Comment (radgupte) [inline src/HomePage/Home.js:11]: You might want to take these closing braces on the next line to keep your code consistent.

**Review by radgupte — APPROVED** [review]
Besides the one change I have suggested above, your code looks good. I'm approving your PR.

File: src/HomePage/Home.js (hunk 1 above)
Comment (sirichandanagangam) [reply src/HomePage/Home.js:11]: Done

**Pull request merged**